### PR TITLE
Test

### PR DESCRIPTION
**Pull request by Alkayde**

### CODE REVIEW

File: src/getHumanAge.test.js [hunk 1]
```diff
@@ -8,6 +8,48 @@ describe(`Function 'getHumanAge':`, () => {
   });
 
   it(`should return an array`, () => {
+    expect(getHumanAge(3, 4)).toBeInstanceOf(Array);
+  });
+
+  it(`should return [0,0] array if catAge<15 and dogAge<15`, () => {
```
Comment (brainwashers) [inline src/getHumanAge.test.js:14]:
```suggestion
  it(`should return [0, 0] array if catAge < 15 and dogAge < 15`, () => {
```

File: src/getHumanAge.test.js [hunk 2]
```diff
@@ -8,6 +8,48 @@ describe(`Function 'getHumanAge':`, () => {
...
+    expect(getHumanAge(3, 4)).toEqual([0, 0]);
+  });
+
+  it(`should return [1,1] if catAge=15 and dogAge=15`, () => {
```
Comment (brainwashers) [inline src/getHumanAge.test.js:18]:
```suggestion
  it(`should return [1, 1] if catAge = 15 and dogAge = 15`, () => {
```

File: src/getHumanAge.test.js [hunk 3]
```diff
@@ -8,6 +8,48 @@ describe(`Function 'getHumanAge':`, () => {
...
+    expect(getHumanAge(15, 15)).toEqual([1, 1]);
+  });
+
+  it(`should return [1,1] if catAge>15, catAge<23, dogAge>15 and dogAge<23`,
```
Comment (brainwashers) [inline src/getHumanAge.test.js:22]: check boundary values

**Review by brainwashers — CHANGES_REQUESTED** [review]
